src/main/java/org/openstreetmap/atlas/geography/atlas/dynamic/DynamicAtlas.java

```diff
@@ -138,6 +138,33 @@ public Rectangle bounds()
         return this.current.bounds();
     }
 
+    public void buildUnderlyingMultiAtlas()
+    {
+        logger.debug("{}: Loading MultiAtlas with {}", this.getName(),
```

Comment:
should this be wrapped in a ```logger.isDebugEnabled()``` block?  No reason to build the shard names if this log message won't get emitted

Reply:
That sounds good!